extensions/divina.md

```diff
@@ -172,7 +179,183 @@ This current draft does not cover guided navigation over alternate versions of e
 ]
 ```
 
-## 5. Package
+## 5. Transitions
+
+In the reading order of a visual narrative, Link Objects <strong class="rfc">may</strong> contain the following additional properties: 
+
+| Key   | Semantics | Type     | Values    | 
+| ----- | --------- | -------- | --------- | 
+| [transitionForward](#transitionForward-transitionBackward) | Describes the transition to be applied when moving FROM the previous resource TO current resource in reading order  | Transition Object | See [Transition Object](#the-transition-object)  | 
+| [transitionBackward](#transitionForward-transitionBackward) | Describes the transition to be applied when moving FROM the current resource TO the previous resource in reading order  | Transition Object | See [Transition Object](#the-transition-object)  | 
+
+A transition disables any continuity between the current resource and the next, i.e. it locally applies a form of `continuous`=`false`. Therefore, a transition represents the only way of creating “breaks” or “pages” in a webtoon (i.e. when continuous=true). However, do note that:
```

Comment:
I find this statement very problematic since we want to make sure that properties are independent from one another.

Transitions should not impact the concept of continuity, it should only affect how discontinuity between two resources is handled.

Reply:
I accept to move this to implementation guidelines to avoid a loong discussion.